From 57ebc9b5c71625dcb89560bed4ffabbb3db53a7e Mon Sep 17 00:00:00 2001
From: mpolacek <mpolacek@138bc75d-0d04-0410-961f-82ee72b054a4>
Date: Mon, 5 May 2014 20:40:33 +0000
Subject: [PATCH] 	PR driver/61065
 	* opts.c (common_handle_option): Call error_at instead of warning_at.

git-svn-id: svn+ssh://gcc.gnu.org/svn/gcc/branches/gcc-4_9-branch@210081 138bc75d-0d04-0410-961f-82ee72b054a4
---
 gcc/opts.c    |    6 +++---
 2 files changed, 11 insertions(+), 3 deletions(-)

diff --git a/gcc/opts.c b/gcc/opts.c
index 3f3db1a..faa63d6 100644
--- a/gcc/opts.c
+++ b/gcc/opts.c
@@ -1501,9 +1501,9 @@ common_handle_option (struct gcc_options *opts,
 		}
 
 	    if (! found)
-	      warning_at (loc, 0,
-			  "unrecognized argument to -fsanitize= option: %q.*s",
-			  (int) len, p);
+	      error_at (loc,
+			"unrecognized argument to -fsanitize= option: %q.*s",
+			(int) len, p);
 
 	    if (comma == NULL)
 	      break;
-- 
1.7.1

